module/type/TypeBrowser.kt

```diff
@@ -98,7 +98,7 @@ class TypeBrowser(isOpen: Boolean = false, order: Int) : Browsers.Browser(isOpen
     private fun exportButton(navState: Navigator.NavigatorState<ThingTypeState<*, *>>) = IconButtonArg(
         icon = Icon.EXPORT,
         enabled = Service.project.current != null && !Service.schema.hasRunningCommand,
-        tooltip = Tooltip.Arg(title = Label.EXPORT_SCHEMA)
+        tooltip = Tooltip.Arg(title = Label.EXPORT_TYPES)
     ) {
         Service.schema.exportTypeSchemaAsync { schema ->
```

Comment:
if you're doing the above rename you shoudl also rename these names (at least `schema` -> `types`)?

Reply:
Nice catch, yes. We've called it `typeSchema`, so I'm following that naming and changing `Label.EXPORT_TYPES` to `Label.EXPORT_TYPE_SCHEMA`.